### ISSUE TITLE
System.NullReferenceException

### ISSUE THREAD

**Issue by okulus** [canton7/SyncTrayzor#318]
```
Version: 1.1.12.0; Variant: Portable; Arch: Amd64
Path: D:\syncthing\SyncTrayzorPortable-x64\SyncTrayzor.exe
System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в SyncTrayzor.Pages.ViewerViewModel.OnClose() в C:\projects\synctrayzor\src\SyncTrayzor\Pages\ViewerViewModel.cs:строка 227
   в Stylet.Screen.<Stylet.IScreenState.Close>b__27(ScreenState oldState, ScreenState newState) в c:\projects\stylet\Stylet\Screen.cs:строка 192
   в Stylet.Screen.SetState(ScreenState newState, Action`2 changedHandler) в c:\projects\stylet\Stylet\Screen.cs:строка 139
   в Stylet.ScreenExtensions.TryClose(Object screen) в c:\projects\stylet\Stylet\ScreenExtensions.cs:строка 40
   в Stylet.ScreenExtensions.<>c__DisplayClass7.<CloseWith>b__6(Object o, CloseEventArgs e) в c:\projects\stylet\Stylet\ScreenExtensions.cs:строка 110
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в Stylet.Screen.<Stylet.IScreenState.Close>b__27(ScreenState oldState, ScreenState newState) в c:\projects\stylet\Stylet\Screen.cs:строка 196
   в Stylet.Screen.SetState(ScreenState newState, Action`2 changedHandler) в c:\projects\stylet\Stylet\Screen.cs:строка 139
   в Stylet.ScreenExtensions.TryClose(Object screen) в c:\projects\stylet\Stylet\ScreenExtensions.cs:строка 40
   в System.Windows.Window.OnClosed(EventArgs e)
   в System.Windows.Window.WmDestroy()
   в System.Windows.Window.WindowFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   в System.Windows.Interop.HwndSource.PublicHooksFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   в MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   в MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   в System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   в System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

[SyncTrayzor.zip](https://github.com/canton7/SyncTrayzor/files/557529/SyncTrayzor.zip)

**Comment by canton7** [comment]
Thanks for reporting! I'll get this fixed.

**Comment by canton7** [comment]
Turns out this has already been fixed as part of #303. I should really release again soon....
